source/fundamentals/crud/write-operations/change.txt

```diff
@@ -0,0 +1,498 @@
+     - | Specifies whether the replace operation bypasses document validation. This lets you 
+         replace documents that don't meet the schema validation requirements, if any 
+         exist. See :manual:`the MongoDB server manual</core/schema-validation/#schema-validation>`
+         for more information on schema validation.
+
+   * - ``Collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. See :manual:`the MongoDB server manual</reference/collation/#std-label-collation>`
+         for more information on collation.
+
+   * - ``Comment``
+     - | Gets or sets the user-provided comment for the operation. 
+         See :manual:`the MongoDB server manual</reference/command/update/#command-fields>`
+         for more information.
+
+   * - ``Hint``
+     - | Gets or sets the index to use to scan for documents. 
+         See :manual:`the MongoDB server manual</reference/command/update/#std-label-update-command-hint>`
+         for more information.
+
+   * - ``IsUpsert``
+     - | Specifies whether the replace operation performs an upsert operation if no 
+         documents match the query filter. 
+         See :manual:`the MongoDB server manual </reference/command/update/#std-label-update-command-upsert>`
+         for more information.
+
+   * - ``Let``
+     - | Gets or sets the let document. 
+         See :manual:`the MongoDB server manual </reference/command/update/#std-label-update-command-upsert>`
+         for more information.
+
+Return Value
+~~~~~~~~~~~~
+
+The ``ReplaceOne()`` method returns a ``ReplaceOneResult`` 
+object. The ``ReplaceOneResult`` type contains the following properties:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``IsAcknowledged``
+     - | Indicates whether the replace operation was acknowledged by MongoDB.
+
+   * - ``IsModifiedCountAvailable``
+     - | Indicates whether you can read the count of replaced records on the
+         ``ReplaceOneResult``.
+
+   * - ``MatchedCount``
+     - | The number of documents that matched the query filter, regardless of
+         whether one was replaced. 
+
+   * - ``ModifiedCount``
+     - | The number of documents replaced by the replace operation. 
+
+   * - ``UpsertedId``
+     - | The ID of the document that was upserted in the database, if the driver
+         performed an upsert.
+
+Example
+~~~~~~~
+
+The following code uses the ``ReplaceOne()`` method to find the first document where the 
+``name`` field has the value "Pizza Town", then replaces this document 
+with a new ``Restaurant`` document named "Food World". Because the ``IsUpsert`` option is 
+set to ``true``, the driver inserts a new document if the query filter doesn't 
+match any existing documents.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: csharp
+
+      var filter = Builders<Restaurant>.Filter.Eq(r => r.Name, "Pizza Town");
+
```

Comment:
[suggestion]
Potentially consider replacing the `r` with `restaurant` to have it match the other code samples and also improve readability a little bit.